content/applications/finance/accounting/reporting/declarations/intrastat.rst

```diff
@@ -0,0 +1,29 @@
+=========
+Intrastat
+=========
+
+**Intrastat** is the system for data collection and statistic production of goods traded among the
+EU Member States. Organizations whose annual trade surpasses a certain threshold must
+provide data of the goods traded with other EU countries.
+
+.. seealso::
+   - `Intrastat
+     <https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Glossary:Intrastat>`_
+
+.. note::
+   As of January 1, 2022, the reporting requirements have changed. One main modification concerns
+   the transactions codes which are now switching to a 2-digits format.
+   All transactions recorded after this date have to follow the new system’s rules.
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Settings --> Customer Invoices`, then enable **Intrastat** and
+*Save*.```
+Once the Intrastat has been activated you are able to choose the **Default incoterm**.
```

Comment:
- why the three ``` ?
- line break at 100th character

```suggestion
Go to :menuselection:`Accounting --> Settings --> Customer Invoices`, then enable **Intrastat** and
*Save*. Once the Intrastat has been activated you are able to choose the **Default incoterm**.
```